### ISSUE TITLE
bitcoin-qt not using the blockchain of bitcoind

### ISSUE THREAD

**Issue by walterkok** [Stadicus/guides#355]
hi, i have everything working well (bitcoind / lnd ) still on testnet. I also installed RTL UI on the PI and that works very well too. Before I move to BTC Mainnet I wanted to test with bitcoin-qt, also installed on the PI. Through the VNC viewer (also use that for RTL) I can startup bitcoin-qt on the PI but it starts downloading its own chain, even if I set the datadir to the same as bitcoind. Bitcoin-qt seems to be using bitcoin.conf apart from the datadir. Any views whether this is fixable or is it always downloading a seperate chain

**Comment by Stadicus** [comment]
bitcoin-qt stores some settings apart from the config file, but it should not be a problem to share the data directory. I don't think this is advisable, though, as my guess is that running bitcoind & bitcoin-qt at the same time (eg. by mistake, just once) could lead to problem.

That being said, you can configure both the data dir and the config file explicitly:
```
-conf=<file> 	  Specify configuration file (default: bitcoin.conf)
-datadir=<dir> 	Specify data directory 
```
https://en.bitcoin.it/wiki/Running_Bitcoin

**Comment by walterkok** [comment]
Thank you Stadicus, I did install everything according to your guide (other than allowing VNC server afterwards) so the following should do the trick to test (would appreciate your validation!) 

As admin:

$ sudo systemctl stop bitcoind
$ sudo systemctl stop lnd

then Open VNC session, in that session a console
$ sudo su - bitcoin
$ cd /usr/local/bin/ #in this location is bitcoin-qt

$ bitcoin-qt -conf=/home/bitcoin/.bitcon/bitcoin.conf -datadir=/home/bitcoin/.bitcon/testnet3/

**Comment by Stadicus** [comment]
Seems legit, except for the `.bitcon` typo... :-)

Did you try it out yet?

**Comment by walterkok** [comment]
Hi Stadicus, I managed to get it working today, the following did the trick:

As admin:

$ sudo systemctl stop bitcoind
$ sudo systemctl stop lnd

then Open VNC session, in that session a console
It is imporant that the VNC session is also logged in as bitcoin otherwise the bitcoin-qt does not start (cannot display its output). So I logged also into the Raspi VNC environment as bitcoin
$ # sudo su - bitcoin # now this is not needed
$ cd /usr/local/bin/ #in this location is bitcoin-qt

$ bitcoin-qt -conf=/home/bitcoin/.bitcoin/bitcoin.conf -datadir=/home/bitcoin/.bitcon
(so no reference to the testnet3 directory). 

Bitcoin-qt started nicely on the Testchain (testnet=1 is in .conf) and I managed to get some testcoins in the wallet from a faucet. Thanks !
